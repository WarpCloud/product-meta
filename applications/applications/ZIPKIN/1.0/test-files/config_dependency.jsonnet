{
  application_type: "ZIPKIN",
  application_name: "zipkin",
  application_version: "1.0",
  user_config: {},
}
