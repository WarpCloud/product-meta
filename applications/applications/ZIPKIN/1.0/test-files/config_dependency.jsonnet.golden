{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "zipkin",
            "application_type": "ZIPKIN"
         },
         "configs": { },
         "instance_settings": { },
         "moduleName": "zipkin",
         "name": "zipkin-zipkin",
         "version": "1.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
